R/zzz.R

```diff
@@ -0,0 +1,17 @@
+.onLoad <- function(...) {
+
+  # Only for x64 bits packages. This is required to avoid error when package is being checked on CI for x86
+  is64 <- (.Machine$sizeof.pointer==8)

```

Comment:
@PavelBal I had to actually do that because package creation would fail. 
It tried to install the package on x86 and loading fails obviously. There might be a better solution. I tried to remove the check from R CMD CHECK but I was getting other erros